ledger-service/http-json-testing/src/main/scala/com/daml/http/HttpServiceTestFixture.scala

```diff
@@ -65,6 +66,22 @@ object HttpServiceTestFixture extends LazyLogging with Assertions with Inside {
 
   private val doNotReloadPackages = FiniteDuration(100, DAYS)
 
+  def withHttpService[A](
```

Comment:
This overload accepts a `testFn` whose arg is a `java.net.URI`, so the caller (in our case `http-json-perf`) doesn't need to build against `akka.http.scaladsl.model.Uri`.